src/main/java/mate/jdbc/dao/impl/CarDaoImpl.java

```diff
@@ -0,0 +1,201 @@
+                    + car + " in driversDB.", e);
+        }
+        deleteDriverByCarId(car.getId());
+        insertDriversForCar(car);
+        return car;
+    }
+
+    @Override
+    public boolean delete(Long id) {
+        String query = "UPDATE cars SET is_deleted = TRUE WHERE id = ?;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            return statement.executeUpdate() > 0;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't delete driver with id " + id, e);
+        }
+    }
+
+    @Override
+    public List<Car> getAllByDriver(Long driverId) {
+        String query = "SELECT c.id AS car_id, model, manufacturer_id, m.id "
+                + "AS manufacture_id, m.name, m.country"
+                + "FROM cars c JOIN cars_drivers ON c.id = cars_drivers.car_id"
+                + " JOIN manufacturers m ON c.manufacturer_id = m.id"
+                + "WHERE  cars_drivers.driver_id = ? AND c.is_deleted = FALSE;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query)) {
+            preparedStatement.setLong(1,driverId);
+            ResultSet resultSet = preparedStatement.executeQuery();
+            List<Car> cars = new ArrayList<>();
+            while (resultSet.next()) {
+                cars.add(getCar(resultSet));
+            }
+            return cars;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get all cars by driver id " + driverId, e);
+        }
+    }
+
+    private Car getCar(ResultSet resultSet) throws SQLException {
+        return new Car(resultSet.getObject("car_id", Long.class),
+                resultSet.getString("model"), getManufacturer(resultSet));
+    }
+
+    private Manufacturer getManufacturer(ResultSet resultSet) throws SQLException {
+        return new Manufacturer(resultSet.getLong("manufacturer_id"),
+                resultSet.getString("name"), resultSet.getString("country"));
+    }
+
+    private List<Driver> getDriversCar(Long id) {
+        String query = "SELECT d.id, d.name, d.license_number FROM drivers d "
+                + "JOIN cars_drivers c_d ON d.id = c_d.driver_id WHERE c_d.car_id = ?;";
+        List<Driver> drivers = new ArrayList<>();
+        try (Connection connection = ConnectionUtil.getConnection();
+                 PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setLong(1, id);
+            ResultSet resultSet = statement.executeQuery();
+            while (resultSet.next()) {
+                drivers.add(getDriver(resultSet));
+            }
+            return drivers;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Couldn't get car by id " + id, e);
```

Comment:
```suggestion
            throw new DataProcessingException("Couldn't get drivers for car by id " + id, e);
```
Recheck all the error messages.